schema/nmdc.yaml

```diff
@@ -1333,6 +1165,15 @@ slots:
   sample_collection_minute:
     range: integer
     
+  salinity_category:
```

Comment:
add a see_also to a mixs ticket requesting this

Reply:
Will do.